frappe/integrations/doctype/ldap_settings/ldap_settings.json

```diff
@@ -1,37 +1,49 @@
 {
+ "actions": [],
  "creation": "2016-09-22 04:16:48.829658",
  "doctype": "DocType",
  "document_type": "System",
  "editable_grid": 1,
  "engine": "InnoDB",
  "field_order": [
   "enabled",
-  "ldap_server_url",
+  "ldap_server_settings_section",
+  "ldap_directory_server",
   "column_break_4",
+  "ldap_server_url",
+  "ldap_auth_section",
   "base_dn",
+  "column_break_8",
   "password",
-  "section_break_5",
+  "ldap_search_and_paths_section",
   "organizational_unit",
```

Comment:
refactor:

As any valid fdn folder path can be used, Rename to `user_search_path` and update interface. Do the same for the groups ou, `group_search_path`

Reply:
implemented in 08e52a60a808caa37e44efa507499785afe4126c

post changes the following commands returned nil from repo path root.
``` bash
grep -r 'organizational_unit_for_groups' .
grep -r 'organizational_unit' .
```